kotlin/internal/lint/ktlint_fix.bzl

```
@@ -15,7 +15,7 @@ def _ktlint_fix_impl(ctx):
 
     # Much of the following is lifted from:
     # https://cs.opensource.google/bazel/bazel/+/refs/tags/4.0.0:src/main/java/com/google/devtools/build/lib/bazel/rules/java/java_stub_template.txt;l=114
```

```suggestion
    # https://github.com/bazelbuild/rules_java/blob/master/java/bazel/rules/java_stub_template.txt
```
might be good to apply too